### ISSUE TITLE
Proxy GET request returns 415

### ISSUE THREAD

**Issue by Blacktiger** [ember-cli/ember-cli#8035]
I'm trying to use the proxy option (ie `ember serve --proxy http://localhost:9000`). Then I have a request from my application that tries to do a simple get (via `axios.get('rest/service')`). Something about the proxied request is causing a 415 UNSUPPORTED MEDIA TYPE but I don't see anything useful in the console output.

```
GET /rest/vulnerabilities 415 9.003 ms - -
```

I have a create-react-app that I was testing that also did a proxy to that same server and used axios to get the data and everything there worked fine so it seems like something with the proxy implementation is amiss maybe?

---

Output from `ember version --verbose && npm --version && yarn --version`:
```
> ember version --verbose
ember-cli: 3.4.1
http_parser: 2.7.0
node: 8.9.0
v8: 6.1.534.46
uv: 1.15.0
zlib: 1.2.11
ares: 1.10.1-DEV
modules: 57
nghttp2: 1.25.0
openssl: 1.0.2l
icu: 59.1
unicode: 9.0
cldr: 31.0.1
tz: 2017b
os: darwin x64

> npm --version
5.5.1
```

**Comment by cibernox** [comment]
Just to give an update on this isuse, the problem is in the live reload implementation.

`ember serve --no-live-reload` can be a temporary workaround.

**Comment by Blacktiger** [comment]
Thanks for the workaround.

**Comment by rwjblue** [comment]
Fixed by #8038 and released in v3.4.2.

**Comment by Blacktiger** [comment]
@rwjblue I'm still seeing this issue after upgrading to v3.4.2.
